src/store/channels/saga.ts

```diff
@@ -23,11 +24,10 @@ export function* joinChannel(action) {
   );
 }
 
-/**
- * Marks all messages as "read" in a channel for a specific user (queries zero-api & sendbird).
- */
-export function* markAllMessagesAsReadInChannel(action) {
-  const { channelId, userId } = action.payload;
+export function* markAllMessagesAsRead(channelId, userId) {
+  if (!channelId || !userId) {
+    return;
+  } // in case of an admin message userId can be undefined
```

Comment:
Isn't the user id supposed to be the currently logged in user? We're saying, "this user has read these messages". So, admin messages shouldn't have any impact here.

We probably still need this in the case of public viewers because they won't have a userId but I'm worried we're passing the wrong user id here maybe?